boskos/metrics/metrics.go

```diff
@@ -58,7 +58,7 @@ func init() {
 }
 
 func main() {
-	logrusutil.ComponentInit("boskos-metrics")
```

Comment:
similar comment - this is a change in behavior.

Reply:
The image is `boskos-metrics`, too -- we should maybe make the binary `boskos-metrics` as well?